src/Core/Constants.cs

```diff
@@ -30,6 +30,7 @@ public static class Constants
         public static string EventCollectionKey = "eventCollection";
         public static string RememberedEmailKey = "rememberedEmail";
         public static string RememberedOrgIdentifierKey = "rememberedOrgIdentifier";
+        public static string PasswordlessLoginNofiticationKey = "passwordlessLoginNofiticationKey";
```

Comment:
typo `PasswordlessLoginNofiticationKey` => `PasswordlessLoginNotificationKey` and also on the string